### PR TITLE
Searchfeeder replacement

### PR DESCRIPTION
**Pull request by BerniVarga**

### CODE REVIEW

File: healthcheck_handler.go [hunk 1]
```diff
@@ -111,7 +111,7 @@ func gtgCheck(handler func() (string, error)) gtg.Status {
 func (h *HealthcheckHandler) checkIfDocumentStoreIsReachable() (string, error) {
 	_, _, err := utils.ExecuteSimpleHTTPRequest(h.docStoreAPIBaseURL+GTGEndpoint, h.httpClient)
 	if err != nil {
-		log.Errorf("Healthcheck: %v", err.Error())
+		logger.Errorf("Healthcheck: %v", err.Error())
```
Comment (tosan88) [inline healthcheck_handler.go]: Just a flavor thing but the error could be reported with `logger.WithError(err).Errorf(...)`

File: processor/model.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+type ContentModel map[string]interface{}
+
+type CombinedModel struct {
+	UUID     string       `json:"uuid"`
+	Content  ContentModel `json:"content"`
+	Metadata []Annotation `json:"metadata"`
+
+	ContentURI    string `json:"contentUri"`
+	LastModified  string `json:"lastModified"`
+	MarkedDeleted string `json:"markedDeleted"`
```
Comment (tosan88) [inline processor/model.go]: Wouldn't better to be a `bool`?

Comment (BerniVarga) [reply processor/model.go]: In some cases, we don't have any knowledge about this content attribute. To support these cases, I think it is more convenient not to use the boolean's false default, but an empty value.

File: healthcheck_handler.go (hunk 1 above)
Comment (BerniVarga) [reply healthcheck_handler.go]: Sure, I will change these. However, it won't be very useful due to the lack of consistency in our projects, when logging error messages.

**Pull request merged**